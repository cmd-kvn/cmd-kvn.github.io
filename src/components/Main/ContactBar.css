nav img {
    height: 60px;
    width: 60px;
    padding: 20px;
}