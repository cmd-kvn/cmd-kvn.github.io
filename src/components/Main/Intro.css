article {
    padding: 8px;
}