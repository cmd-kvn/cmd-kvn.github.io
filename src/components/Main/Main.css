main {
    float: left;
    width: 67%;
    height: 100vh;
}