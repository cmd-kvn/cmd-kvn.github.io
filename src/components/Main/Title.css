h1 {
    font-size: 72px;
    font-family: Rock Salt;
    margin-top: 60px
}

h2 {
    font-size: 24px;
    font-family: Cabin;
    /*margin-top: 100px;*/
}
