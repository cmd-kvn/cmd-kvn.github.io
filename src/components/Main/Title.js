import React from 'react';

const Title = () => {
    let fontSize = {
        fontSize: 72
    };

    return (
        <div>
            <p>
                <span style={fontSize}>Kevin Wong</span> <br />
                Full Stack Web Developer &nbsp; - &nbsp; React.js / Node.js
            </p>
            <hr />
        </div>
    );
};

export default Title;
