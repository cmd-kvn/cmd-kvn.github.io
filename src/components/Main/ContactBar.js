import React from 'react';

import './ContactBar.css';

const ContactBar = () => {
    return (
        <nav>
            <img src='email-icon.png' alt='email icon' />
            <img src='github-icon.png' alt='github icon' />
            <img src='linkedIn-icon.png' alt='linkedIn icon' />
        </nav>
    );
};

export default ContactBar;
