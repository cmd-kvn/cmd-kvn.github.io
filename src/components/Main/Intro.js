import React from 'react';
import Markdown from 'markdown-to-jsx';

import './Intro.css';

const markdown = `
## Hello! I'm Kevin - your friendly neighborhood JavaScript developer.  
When I'm not protecting the community from \`eslint\` errors and 
unresolved \`Promise\` chains you will find me volunteering, at your 
local Meetup, or dancing Cuban salsa!
`.trim();

const Intro = () => {
    return (
        <article>
            <Markdown>
                {markdown}
            </Markdown>
        </article>
    );
};

export default Intro;
