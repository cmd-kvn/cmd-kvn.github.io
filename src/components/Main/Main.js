import React from 'react';

import Title from './Title';
import Intro from './Intro';
import ContactBar from './ContactBar';
// import '.css';

const Main = () => {
    return (
        <main>
            <Title />
            <Intro />
            <ContactBar />
        </main>
    );
};

export default Main;
