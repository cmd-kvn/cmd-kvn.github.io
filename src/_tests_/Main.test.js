// /* eslint-disable */
import React from 'react';
// import ReactDOM from 'react-dom';
// import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Main from '../components/Main/Main';

describe('Main', function() {
    it('runs the Main', function() {
        const rendered = shallow(<Main />);
        expect(toJson(rendered)).toMatchSnapshot();
    });
});

// TODO: investigate/disable these linting errors and
// determine which npm packages to actually use
