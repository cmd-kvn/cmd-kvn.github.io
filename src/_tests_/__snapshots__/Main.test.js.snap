exports[`Main runs the Main 1`] = `
<main>
  <Title />
  <Intro />
  <ContactBar />
</main>
`;
